mod-patron-blocks/src/main/resources/domain/mod-patron-blocks/features/samples/patron-notice-policy-entity.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id" : "ed2ee745-889c-4163-911b-2b73a4ee51b9",
+  "name" : "firstPolicy",
+  "description" : "description",
+  "loanNotices" : [ ],
+  "feeFineNotices" : [ ],
+  "requestNotices" : [
+    {
+      "name" : "Test request name",
+      "templateId" : "2d5f138f-e088-4dff-80f6-9830f13bcde7",
+      "templateName" : "Test template name",
+      "format" : "Email",
+      "frequency" : "One time",
+      "realTime" : true,
+      "sendOptions" : {
+        "sendHow" : "Before",
+        "sendWhen" : "Recall request"
```

Comment:
This request notice configuration looks odd. It can't have a `name` and `templateName`. Notice with triggering event `Recall request` can't have `frequency`, as well as timing `Before`. Here is an example of patron notice policy with one notice for `Recall request`:
```
{
  "name": "request notices",
  "requestNotices": [{
      "templateId": "62e51794-6a49-41a6-9684-72c23ce66f63",
      "format": "Email",
      "sendOptions": {
        "sendWhen": "Recall request"
      },
      "realTime": false
    }
  ],
  "id": "b8669554-7edf-4099-a668-c0fa2e3af3d9"
}
```